bin/echo.c

```diff
@@ -177,31 +160,33 @@ int negotiate(struct s2n_connection *conn, int fd)
     printf("Curve: %s\n", s2n_connection_get_curve(conn));
     printf("KEM: %s\n", s2n_connection_get_kem_name(conn));
     printf("KEM Group: %s\n", s2n_connection_get_kem_group_name(conn));
+    printf("Cipher negotiated: %s\n", s2n_connection_get_cipher(conn));
+
+    if (session_resumed) {
+        printf("Resumed session\n");
+    }
```

Comment:
Why was the order of the prints changed? Some consumers might rely on the order so we shouldn't change it unless there is a reason to.